packages/plugins/users-permissions/server/controllers/auth.js

```diff
@@ -367,8 +361,11 @@ module.exports = {
     } catch (err) {
       if (_.includes(err.message, 'username')) {
         throw new ApplicationError('Username already taken');
-      } else {
+      } else if (_.includes(err.message, 'email')) {
```

Comment:
this is another bug I noticed.
Other errors coming from the service would be wrongly changed here to "Email already taken" although the actual error is something else. Example: when E-Mail sending fails